view.rb

```diff
@@ -19,4 +19,13 @@ def ask_for_name
     puts "What is the animal's name?"
     return gets.chomp
   end
-end
\ No newline at end of file
+
+  def ask_for_animal_index
+    puts 'Which animal would you like to adopt? Key in the index number.'
+    return gets.chomp.to_i
+  end
+
+  def adopted_animal(animal)
+    puts "Thank you! Please take care of #{animal}!"
```

Comment:
awwww 😻

Reply:
Hi Rabea! Thanks for the tips on improving the new feature, have edited where necessary.

Reply:
That was quick! Great work!